docs/source/API/core/parallel-dispatch/parallel_for.rst

```diff
@@ -0,0 +1,127 @@
+``parallel_for()``
+==================
+
+.. role::cpp(code)
+    :language: cpp
+
+Header File: ``<Kokkos_Core.hpp>``
+
+Usage:
+
+.. code-block:: cpp
+
+    Kokkos::parallel_for(name, policy, functor);
+    Kokkos::parallel_for(policy, functor);
+
+.. _text: ../policies/ExecutionPolicyConcept.html
+
+.. |text| replace:: *ExecutionPolicy*
+
+Dispatches parallel work defined by ``functor`` according to the |text|_ ``policy``. The optional label ``name`` is
+used by profiling and debugging tools. This call may be asynchronous and return to the callee immediately. 
+
```

Comment:
I know we're not dealing with content at this time, but the summary should also include `KOKKOS_LAMBDA`, in addition to the already mentioned `functor`.